wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Create/ConversationCreationController.swift

```diff
@@ -274,13 +281,13 @@ extension ConversationCreationController: AddParticipantsConversationCreationDel
             values.participants = users
 
         case .create:
-            guard let userSession = ZMUserSession.shared() else { return }
+            guard let userSession = userSession as? ZMUserSession else { return }
```

Comment:
What's the reason why we don't inject the userSession here as well and instead we safely unwrap?

Reply:
Same reason as below

Reply:
Can we add a TODO comment so we can fix it on a later stage?

Reply:
done https://github.com/wireapp/wire-ios/pull/729/commits/3c218a48044ccf7e46d88cc0db69242c3904a8f6